core/src/main/kotlin/org/evomaster/core/problem/security/service/HttpCallbackVerifier.kt

```diff
@@ -75,7 +80,9 @@ class HttpCallbackVerifier {
      * Method generates a unique callback link to be used as payload for SSRF.
      */
     fun generateCallbackLink(name: String): String {
-        val ssrfPath = "/sink/${counter++}"
+        // FIXME: sink/EM_0 <- slash get replaced with a comma at some point, which fails
```

Comment:
This was a problem too, for some reason slash after sink get replaced with a comma when recomputing the fitness.